src/modules/clarAllTasks.js

```diff
@@ -0,0 +1,22 @@
+const toggleFinishedTask = (e) => {
+  e.nextElementSibling.classList.toggle('line_through');
+  e.nextElementSibling.classList.toggle('change_color');
+  const taskList = JSON.parse(localStorage.getItem('toDoList'));
+  taskList[e.parentElement.id].completed = !taskList[e.parentElement.id].completed;
+  localStorage.setItem('toDoList', JSON.stringify(taskList));
+};
+
+const clearSelectedTasks = () => {
+  let taskList = JSON.parse(localStorage.getItem('toDoList'));
+  taskList = taskList.filter((completed) => completed.completed !== true);
+  let i = 0;
+  taskList.forEach((task) => {
+    task.index = i;
+    // eslint-disable-next-line no-plusplus
+    i++;
+  });
```

Comment:
- [ ] Kindly make sure you fix your linters errors instead of disabling them, this is for best practices and clean code. To fix this you can change `i++` to `i += 1` as both are same.